### PR TITLE
Add utilities section in docs and change svg for organization-status

### PR DESCRIPTION
**Pull request by Unetitebiere**
-remove text spacing with class caps
+add section for utilities in Typography
+fix icon name

### CODE REVIEW

File: docs/_sass/layout/_styleguide.scss [hunk 1]
```diff
@@ -25,6 +25,13 @@
   padding: 1.5em;
 }
 
+//text
+div {
+  .break-word, .truncate {
+    width: 145px;
+  }
```
Comment (marilynechan) [inline docs/_sass/layout/_styleguide.scss]: I would add inline style in the documentation instead, to demonstrate the utility of the class.

File: docs/_components/typography/03_text-utility.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <div class="right-align">
+        Text in right-align
+    </div>
+
+    <div class="justify">
+        Text in justify
+    </div>
+
+    <div class="nowarp">
+        Text with nowarp
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Shouldn't it be `nowrap` ?

File: docs/_components/typography/03_text-utility.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+type: typography
+title: Utilities
+usage: Change typographic weights, styles, and alignment with these utility styles.
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: *utility classes.

File: docs/_components/typography/03_text-utility.html [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <div class="italic">
+        Text in italic
+    </div>
+
+    <div class="caps">
+        Text in caps
+    </div>
+
+    <div class="left-align">
+        text in left-align
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Small detail, but should be `Text` with an uppercase T.
Also, it reads weird "Text _in_ left-align". With? With the left-align class?

File: docs/_components/typography/03_text-utility.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+
+    <div class="center">
+        text in center
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Uppercase T here.

File: docs/_components/typography/03_text-utility.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    <div class="nowarp">
+        Text with nowarp
+    </div>
+
+    <div class="break-word">
+        Text with break-word
+    </div>
+
+    <div class="truncate">
+        Text with truncate is not cut
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Hmm, shouldn't this read "is cut"?

File: docs/_components/typography/03_text-utility.html (hunk 2 above)
Comment (wfortin) [inline docs/_components/typography/03_text-utility.html]: +1

**Comment by wfortin** [comment]
I don't have much more comments than @marilynechan, apply those then :shipit:

File: docs/_components/typography/03_text-utility.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+
+    <ul class="list-reset">
+        <li>list reset item 1</li>
+        <li>item 2</li>
+        <li>item 3</li>
+    </ul>
+
+    <div class="pull-right">
+        text pull-right
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Uppercase T.
@wfortin We currently have 4 classes in the styleguide that do this (float right/left). Which should we keep, which should we document?

File: docs/_components/typography/03_text-utility.html [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+    </div>
+
+    <div class="pull-left">
+        text pull-left
```
Comment (marilynechan) [inline docs/_components/typography/03_text-utility.html]: Uppercase T.

**Comment by marilynechan** [comment]
Ship it!
One last comment: I'm not sure about the `organization-status` icon name. I know I asked for the change in your other PR, but seeing it here makes me realize it's really specific to our application... Maybe 'status' simply? Depending on what the icon represents...

File: docs/_components/typography/03_text-utility.html (hunk 7 above)
Comment (wfortin) [inline docs/_components/typography/03_text-utility.html]: `left` and `right` should be the ones used.  (for float left and right)
Please drop `pull-left` and `pull-right`.

Also: these are not "typograpgy" modifiers, we have `align-left` and `align-right` that align the text.

**Comment by marilynechan** [comment]
Great work BTW!

File: docs/_components/typography/03_text-utility.html (hunk 7 above)
Comment (wfortin) [inline docs/_components/typography/03_text-utility.html]: Document `align-left` and `align-right`

**Comment by Unetitebiere** [comment]
@marilynechan we have svg with status-** name, I suggest something like status-circle or status-scan ( because the svg looks like a scan :) ). I think status is a bit too generics

**Pull request merged**